draft-ietf-trans-rfc6962-bis.md

```diff
@@ -2024,26 +2024,33 @@ otherwise considered suitable for use as a cryptographic signature algorithm.
 IANA is asked to establish a registry of `VersionedTransType` values, named
 "CT VersionedTransTypes", that initially consists of:
 
-|-------+---------------------------|
-| Index | Type and Version          |
-|-------+---------------------------|
-| 0     | reserved                  |
-| 1     | x509_entry_v2             |
-| 2     | precert_entry_v2          |
-| 3     | x509_sct_v2               |
-| 4     | precert_sct_v2            |
-| 5     | signed_tree_head_v2       |
-| 6     | consistency_proof_v2      |
-| 7     | inclusion_proof_v2        |
-| 8     | x509_sct_with_proof_v2    |
-| 9     | precert_sct_with_proof_v2 |
-| 65535 | reserved                  |
-|-------+---------------------------|
+|-----------------+---------------------------+------------------------------------------|
+| Value           | Type and Version          | Reference / RFC5226 Assignment Policy    |
+|-----------------+---------------------------+------------------------------------------|
+| 0x0000          | Reserved                  | [RFC6962] (*)                            |
+| 0x0001          | x509_entry_v2             | RFCXXXX                                  |
```

Comment:
Replace RFCXXXX with "This Document" ? 
Not sure we'll get a chance to replace the placeholder with the actual number.